webhook_features/src/features/summary_comment.rs

```diff
@@ -286,23 +283,20 @@ impl AckType {
     }
 }
 
-macro_rules! multi_vec {
-    ($([$($key:literal),+] => $value:expr);*) => {
-        vec![
-            $($(($key, $value)),*),*
-        ]
-    };
-}
-
 lazy_static! {
-    static ref ACK_PATTERNS: Vec<(&'static str, AckType)> = multi_vec![
-        ["code review ack", "cr ack", "cr-ack", "crack"] => AckType::Ack;
-        ["ack", "utack", "tack"] => AckType::Ack;
-        ["concept ack", "concept-ack", "conceptack", "ack"] => AckType::ConceptAck;
-        ["concept nack", "concept-nack", "conceptnack"] => AckType::ConceptNack;
-        ["approach ack", "approach-ack", "approachack"] => AckType::ApproachAck;
-        ["approach nack", "approach-nack", "approachnack"] => AckType::ApproachNack;
-        ["nack"] => AckType::ConceptNack
+    static ref ACK_PATTERNS: Vec<(&'static str, AckType)> = vec![
+        // approach ACK, approach-ack, approachack
+        (r".*\b(Approach ACK)\b.*", AckType::ApproachAck),
+        // approach NACK, approach-nack, approachnack
+        (r".*\b(Approach NACK)\b.*", AckType::ApproachNack),
+        // NACK
+        (r".*\b(NACK)\b", AckType::ConceptNack),
+        // concept ACK, concept-ack, conceptack, ack
+        (r".*\b(Concept ACK)\b.*", AckType::ConceptAck),
```

Comment:
Can remove this case?

Reply:
If I remove the case, comments such as `Concept ACK 1234567890123456789012345678901234567890` will be parsed as ACK, probably not desirable.